### PR TITLE
Fix getMenuProps downshift error

### PR DESCRIPTION
**Pull request by stphnnnn**
- fixes `getMenuProps` downshift error

### CODE REVIEW

File: src/components/Sidebar.js [hunk 1]
```diff
@@ -115,11 +120,8 @@ const Sidebar = ({
         <h2 id="heading-add">
           <VisuallyHidden>Add a Loo</VisuallyHidden>
         </h2>
-        <Box
-          as={NavLink}
+        <StyledNavLink
```
Comment (falconx) [inline src/components/Sidebar.js:123]: out of interest, was `as={NavLink}` causing problems or was it changed due to preference?

Comment (stphnnnn) [reply src/components/Sidebar.js:123]: it was causing problems, the alignItems prop was being applied as an attribute and my browser was complaing

**Comment by stphnnnn** [comment]
> 
> 
> \o/ and fixes an issue where the listbox was referencing a missing id.
> 
> (minor) there is still an issue where we have no results, since the `listbox` role still expects direct children to be role `option`.

I thought this too but I couldn't actually find that specified anywhere. Either way, it feels like a small(ish) concession.

**Comment by falconx** [comment]
> I thought this too but I couldn't actually find that specified anywhere. Either way, it feels like a small(ish) concession.

You can see the issue by displaying no results and running the axe developer tools. But yeah it's definitely minor.

**Pull request merged**